pr_agent/tools/pr_reviewer.py

```diff
@@ -96,7 +96,7 @@ def _prepare_pr_review(self) -> str:
                 del data['PR Feedback']['Security concerns']
                 data['PR Analysis']['Security concerns'] = val
 
-        if settings.config.git_provider == 'github' and \
+        if settings.config.git_provider != 'bitbucket' and \
```

Comment:
**Suggestion:** Consider using a constant for 'bitbucket' instead of a string. This will prevent potential bugs due to typos and make the code easier to maintain.
```suggestion
        BITBUCKET = 'bitbucket'
        ...
        if settings.config.git_provider != BITBUCKET and ...
```